FlightReservation/src/screens/LogIn.js

```diff
@@ -5,35 +5,40 @@ import {Formik} from 'formik';
 import TextField from '../components/form/TextField';
 import PasswordField from '../components/form/PasswordField';
 import {logInValidationSchema} from '../schemas/logInSchema';
-import {signInWithNameEmailAndPassword} from '../helpers/firebaseSignUp';
-import colors from '../consts/colors';
+// import {signInWithNameEmailAndPassword} from '../helpers/firebaseSignUp';
 import style from './../consts/style';
 import {Pressable} from 'react-native';
+import {logInWithEmailAndPassword} from './../helpers/firebaseLogIn';
 
 const LogIn = function ({navigation}) {
   const [loading, setLoading] = useState(false);
+  const [emailInUseError, setEmailInUseError] = useState(false);
 
   //Change the handleSignIn function for the handleLogIn
-  const handleSignIn = values => {
+  const handleLogIn = async values => {
     const {email, password} = values;
     setLoading(true);
-    signInWithNameEmailAndPassword(email, password)
-      .then(() => setEmailInUseError(false))
-      .catch(() => setEmailInUseError(true))
-      .finally(() => setLoading(false));
+    await logInWithEmailAndPassword(email, password).then(response => {
+      setEmailInUseError(false);
+      setLoading(false);
+    });
+    // .catch(() => setEmailInUseError(true))
+    // .finally(() => setLoading(false));
   };
 
   return (
     <SafeAreaView>
       {loading && <Text>Loading...</Text>}
+      {emailInUseError && <Text>Cannot log in</Text>}
+
       <Formik
         validationSchema={logInValidationSchema}
         initialValues={{
           email: 'juan1@example.com',
           password: 'Juan123%',
         }}
```

Comment:
use data from inputs to test the function read the documentation to use formik

Reply:
also you have merge conflicts pleas pull your main branch and solve the conflicts